redis.conf

```diff
@@ -67,8 +67,8 @@ tcp-backlog 511
 # incoming connections. There is no default, so Redis will not listen
 # on a unix socket when not specified.
 #
-# unixsocket /tmp/redis.sock
-# unixsocketperm 700
+unixsocket /tmp/redis.sock
```

Comment:
I would prefer not to modify the default settings. Just add new flags with their default values (lines 507-518) but commented out.